src/PipelinesMicroserviceCLI/Commands/RegisterPipeline.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace PipelinesMicroserviceCLI\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use PipelinesMicroservice\Services\PipelineApi;
+use GuzzleHttp\Client;
+use PipelinesMicroservice\PipelinesMicroserviceApi;
+
+class RegisterPipeline extends CLICommand
+{
+    protected function configure()
+    {
+        $this
+        ->setName('pipeline:register')
+        ->setDescription('Register a pipeline in the microservice')
+        ->addArgument(
```

Comment:
I think we should put this in a configuration file instead as it doesn't change in an installation of this library.